app/client/components/helpCentre/helpCentreArticle.tsx

```diff
@@ -0,0 +1,131 @@
+import { navigate, RouteComponentProps } from "@reach/router";
+import * as Sentry from "@sentry/browser";
```

Comment:
I think we do this in a few places, but it looks like we only use `captureMessage` and `captureException`, can we just import those specifically instead of everything?